wandb/util.py

```diff
@@ -1934,3 +1934,21 @@ def merge_dicts(source: Dict[str, Any], destination: Dict[str, Any]) -> Dict[str
             else:
                 destination[key] = value
     return destination
+
+
+def load_sweep_config(sweep_config_path: str) -> Optional[Dict[str, Any]]:
```

Comment:
why is it in the general util? seems more appropriate to put it under launch and or sweeps?

Reply:
Well its being used in load the sweep config in the **normal** case in addition to the launch case.